workflow-templates/im-deploy-on-prem-website.yml

```diff
@@ -309,54 +313,54 @@ jobs:
 
       - name: Zip the published app for faster deployment and uploads
         working-directory: ${{ env.UNZIPPED_ASSET }}
-        run: zip -r ../${{env.DEPLOY_ZIP}} .
+        run: 7z a -tzip ../${{ env.DEPLOY_ZIP }} .
 
       # Uncomment this if the site hasn't been created before
       # This action does work better if the website cert is already installed
       # - name: Create Web Site
       #   uses: im-open/iis-site-create@v2.0.0
       #   with:
-      #     server: '${{ needs.set-vars.outputs.IIS_SERVER }}'
+      #     server: '${{ needs.set-vars.outputs.ON_PREM_WINDOWS_SERVER }}'
       #     website-name: '${{ needs.set-vars.outputs.WEBSITE_NAME }}'
       #     app-pool-name: '${{ needs.set-vars.outputs.APP_POOL_NAME }}'
       #     website-host-header: '${{ needs.set-vars.outputs.WEBSITE_HOST_HEADER }}'
       #     website-path: '${{ needs.set-vars.outputs.WEBSITE_PATH }}'
       #     website-cert-path: '${{ needs.set-vars.outputs.WEBSITE_CERT_PATH }}'
       #     website-cert-friendly-name: '${{ needs.set-vars.outputs.WEBSITE_CERT_FRIENDLY_NAME }}'
       #     website-cert-password: '${{ needs.set-vars.outputs.WEBSITE_CERT_PASSWORD }}'
-      #     service-account-id: '${{ needs.set-vars.outputs.IIS_SERVER_SERVICE_USER }}'
-      #     service-account-password: '${{ needs.set-vars.outputs.IIS_SERVER_SERVICE_PASSWORD }}'
+      #     service-account-id: '${{ needs.set-vars.outputs.ON_PREM_DEPLOYMENT_SA_ID }}'
+      #     service-account-password: '${{ needs.set-vars.outputs.ON_PREM_DEPLOYMENT_SA_SECRET }}'
```

Comment:
You probably need to restore the `IIS_SERVER_SERVICE_USER` and `IIS_SERVER_SERVICE_PASSWORD` so you can use them here. You certainly don't want the websites to be running with the permissions of the deployment service account.